style/style.css

```diff
@@ -0,0 +1,169 @@
+body {
+  flex-direction: column;
+  align-items: center;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+  margin-top: -40px;
+  margin-left: 0;
+  margin-right: 0;
+  cursor: pointer;
+  width: 100%;
+  justify-content: center;
+  overflow-x: hidden;
+}
+
+.flex {
+  display: flex;
+}
+
+.nav-bar {
+  position: relative;
+  flex: 1 1 11.11%;
+  top: 150px;
+  width: 990px;
+  height: 40px;
+  border-radius: 3px;
+  box-shadow: rgba(56, 56, 56, 1) 0 1px 2px;
+  background: linear-gradient(to top, rgb(120, 120, 120) 17px, rgb(100, 100, 100) 17px);
+  align-items: stretch;
+  z-index: 1;
+  justify-items: center;
+}
+
+.btn-item {
+  position: relative;
+  width: 11.12%;
+  height: 35px;
+  padding-top: 0.5em;
+  margin: auto -1px;
+  background-color: transparent;
+  border: none;
+  border-right: 1px solid rgb(66, 66, 66);
+  color: white;
+  font-weight: 400;
+  text-align: center;
+}
+
+.btn-item:hover {
+  background: linear-gradient(#000, #444, #444, #444, #000);
+}
+
+a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.search {
+  align-items: center;
+  width: 10%;
+  height: 15px;
+  margin: auto 10px;
+  border-radius: 15px;
+  padding: 3px;
+  color: #fff;
+  font-size: 13px;
+  box-shadow: 0 2px 1px rgba(0, 0, 0, 0.6) inset, rgba(255, 255, 255, 0.2) 0 1px 0;
+}
+
+input {
+  width: 70%;
+  margin-left: 3px;
+  border: none;
+  background-color: transparent;
+}
+
+.main-section {
+  background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 62% 100% no-repeat;
+  background-color: transparent;
+  background-size: 1350px 1315px;
+  background-position: center bottom;
+  width: 100%;
+  margin-top: -3px;
+  text-align: center;
+  position: relative;
+  padding-top: 200px;
+  padding-bottom: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  justify-self: flex-start;
+  color: #fff;
+  font-weight: lighter;
+}
+
+.img3 {
+  margin-top: 40px;
+  height: 30px;
+  width: 600px;
+  font-size: 22px;
+  font-weight: lighter;
+}
+
+.grid {
+  width: 1350px;
+  height: 210px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin: 2px auto;
+}
+
+.grid > a > img {
+  width: 100%;
+}
+
+.footer {
+  display: grid;
+  width: 1300px;
+  height: 90px;
+  grid-template-columns: 35% 38%;
+  grid-template-rows: 37% 37%;
+  justify-content: center;
+  align-content: center;
+  font-size: 9px;
+  opacity: 0.7;
+}
+
+.footer-item {
+  text-align: start;
+  align-items: center;
+  width: 1300px;
+  justify-content: center;
+  padding-top: 15px;
+}
+
+.footer-item > ul {
+  list-style: none;
+  margin-left: -40px;
+  margin-top: -5px;
+}
+
+.footer-item > ul > li {
+  display: inline;
+  margin: auto 3px;
+}
+
+ul > li > a {
+  padding-right: 3px;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Thanks @bertil291utn . We fixed the issues